src/main/java/uk/nhs/digital/nhsconnect/nhais/service/SequenceService.java

```diff
@@ -4,19 +4,21 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import uk.nhs.digital.nhsconnect.nhais.exceptions.SenderAndRecipientValidationException;
+import uk.nhs.digital.nhsconnect.nhais.exceptions.SenderValidationException;
 import uk.nhs.digital.nhsconnect.nhais.repository.SequenceRepository;
 
 @Service
 public class SequenceService {
-    private final static String TRANSACTION_ID = "transaction_id";
+    private final static String TRANSACTION_KEY_FORMAT = "transaction_id-%s";
```

Comment:
Can we use 'TN' instead of 'transaction_id' as the prefix?This is the abbreviation used by gp links similar to SIS==Send Interchange Sequence etc.

Reply:
Bartosz already wrote about it, I have changed it :)